src/LaunchDarkly.ServerSdk.Redis/RedisFeatureStoreBuilder.cs

```diff
@@ -24,7 +24,6 @@ internal RedisFeatureStoreBuilder()
         {
             RedisConfig.EndPoints.Add(RedisComponents.DefaultRedisEndPoint);
             RedisConfig.ConnectTimeout = (int)RedisComponents.DefaultConnectTimeout.TotalMilliseconds;
-            RedisConfig.ResponseTimeout = (int)RedisComponents.DefaultResponseTimeout.TotalMilliseconds;

```

Comment:
Doesn't the `SyncTimeout` property provide the functionality we want for this? If so, we shouldn't completely remove the option.

Reply:
I think we probably would want to have some support for setting the sync timeout. Based on https://github.com/StackExchange/StackExchange.Redis/issues/235, I thought these two configuration options were different in nature, so I reasoned that we could add support for `SyncTimeout` in a future PR, but I can see it also making sense to add here since we're going to a major version bump.

Reply:
Oh, wait - we already have a `WithOperationTimeout` setter in the builder that sets `SyncTimeout`. We just didn't call it `SyncTimeout`, and we didn't mention it in this constructor because there's no default value.